.gitignore

```diff
@@ -14,3 +14,4 @@ newspeak
 tools/kompos/node_modules
 tools/kompos/out
 traces
+/npm-debug.log
```

Comment:
Please remove the leading `/`, then it matches `npm-debug.log` files everywhere I think, which would be a bit more useful.